pkg/app/metadata.json

```diff
@@ -0,0 +1,963 @@
+      "examples": [{
+        "cmd": "fastly acl delete --name robots --version latest",
+        "title": "Delete an ACL from the specified service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#delete-acl"]
+    },
+    "describe": {
+      "examples": [{
+        "cmd": "fastly acl describe --name robots --version latest",
+        "title": "Retrieve a single ACL by name for the version and service"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#get-acl"]
+    },
+    "list": {
+      "examples": [{
+        "cmd": "fastly acl list --version latest",
+        "title": "List ACLs for the version and service"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#list-acls"]
+    },
+    "update": {
+      "examples": [{
+        "cmd": "fastly acl update --name robots --new-name blocklist --version latest",
+        "title": "Update an ACL for a particular service and version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#update-acl"]
+    }
+  },
+  "acl-entry": {
+    "create": {
+      "examples": [{
+        "cmd": "fastly acl-entry create --acl-id 50CCynNwD3BjaGJjMTFB1j --ip 64.18.0.0",
+        "title": "Add an ACL entry to an ACL"
+      },
+      {
+        "cmd": "fastly acl-entry create --acl-id 50CCynNwD3BjaGJjMTFB1j --ip 64.18.0.1 --negated",
+        "title": "Add a negated ACL entry to an ACL"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#create-acl-entry"]
+    },
+    "delete": {
+      "examples": [{
+        "cmd": "fastly acl-entry delete --acl-id 50CCynNwD3BjaGJjMTFB1j --id 4DiuYrv9nVoa4HFmQmujT1",
+        "title": "Delete an ACL entry from a specified ACL"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#delete-acl-entry"]
+    },
+    "describe": {
+      "examples": [{
+        "cmd": "fastly acl-entry describe --acl-id 50CCynNwD3BjaGJjMTFB1j --id 4bgjM6X1gb3670gJMEqdKX",
+        "title": "Retrieve a single ACL entry"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#get-acl-entry"]
+    },
+    "list": {
+      "examples": [{
+        "cmd": "fastly acl-entry list --acl-id 50CCynNwD3BjaGJjMTFB1j",
+        "title": "List ACLs from a specified ACL"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#list-acl-entries"]
+    },
+    "update": {
+      "examples": [{
+        "cmd": "fastly acl-entry update --acl-id 50CCynNwD3BjaGJjMTFB1j --id 4bgjM6X1gb3670gJMEqdKX --negated",
+        "title": "Update a specific ACL entry"
+      },
+      {
+        "cmd": "fastly acl-entry update --acl-id 50CCynNwD3BjaGJjMTFB1j --file ./batch.json",
+        "description": "Update multiple ACL entries using JSON batch file. Refer to https://developer.fastly.com/reference/api/acls/acl-entry/#bulk-update-acl-entries for example structure.",
```

Comment:
I think you could assume that the title and description are markdown and I can parse it as such in devhub.